code/game/objects/items.dm

```diff
@@ -196,6 +196,8 @@
 		if(src == user.get_inactive_hand())
 			if(src.flags & TWOHANDABLE)
 				return src.wield(user)
+			if(!user.put_in_hand_check(src, user.get_active_hand()))
```

Comment:
I'm using the code
```
if(W.cant_drop) //if the item can't be dropped
 	var/I = is_holding_item(W) //AND the item is currently being held in one of the mob's hands
 	if(I)
 		to_chat(src, "You can't pry \the [W] out of your [get_index_limb_name(I)]!")
 		return 0
```
to prevent people from grabbing an un-droppable item out of one hand with their other hand.
That code was placed in `put_in_hand_check()`. But since `/obj/item/attack_hand()` calls `u_equip()` before `put_in_hand()`, the item leaves the mob's hands before `put_in_hand()` can call 'put_in_hand_check()`. So I'm manually calling it here, since the alternative would be to copy-paste the above block of code into this location.